app/controllers/verify/review_controller.rb

```diff
@@ -71,7 +70,7 @@ def idv_params
 
     def phone_confirmation_required?
       idv_params[:phone] != current_user.phone &&
```

Comment:
Derp. Sorry. What I meant was only when comparing the values in the hash, not the keys. Because we call `.with_indifferent_access` on the hash, we can look up a value based on either symbol or string keys, which is why we can read `idv_params[:phone]`. However, if `idv_params[:phone]` was assigned a symbol value, the value itself will always be a String, which is why I had to change the comparison in `address_verification_mechanism == 'usps'` to a String, even though I didn't change the assignment to a symbol value (`idv_session.address_verification_mechanism = :usps`).